File: ERPServer/Shared/Shared/Entities/Setup/RepresentiveSearchDTO.cs
using Data.Entities.Shared;
using Shared.Entities.Shared;
using Shared.Enums;

namespace Shared.Entities.Setup
{
    public class RepresentiveSearchDTO : Paging
    {
        public string FullName { get; set; }
        public string AddressDetails { get; set; }
        public string Mobile { get; set; }
        public string Notes { get; set; }
        public RepresentiveTypeEnum? RepresentiveTypeId { get; set; }

    }
}

File: erp-app/src/app/modules/setup/models/representive-search.dto.ts
import { RepresentiveTypeEnum } from "src/app/shared/enums/representive-type.enum";
import { PagingDTO } from "src/app/shared/models/paging-dto";

export class RepresentiveSearchDTO extends PagingDTO {
    isActive: boolean;
    fullName: string;
    addressDetails: string;
    mobile: string;
    notes: string;
    representiveTypeId: RepresentiveTypeEnum | null;
}